design/components/SDCard.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div class="flex flex-row px-2 lg:flex-col">
```

Comment:
You could have used semantic HTML here. E.g. `<article>` 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article